packages/commonwealth/client/scripts/helpers/feature-flags.ts

```diff
@@ -32,6 +32,7 @@ const featureFlags = {
   referrals: buildFlag(process.env.FLAG_REFERRALS),
   stickyEditor: buildFlag(process.env.FLAG_STICKY_EDITOR),
   newMobileNav: buildFlag(process.env.FLAG_NEW_MOBILE_NAV),
+  rewardsPage: buildFlag(process.env.FLAG_REWARDS_PAGE),
```

Comment:
will this be launched as a separate feature or will be needed with both xp/quests as each of those are rolled out?

Reply:
for now this is separate flag but this should be adjusted to be accessible whenever the first feature goes out (referrals/quests)